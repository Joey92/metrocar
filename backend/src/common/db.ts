import pgp from 'pg-promise';
import * as config from './config';

export const pg = pgp();
export const db = pg({
  max: 200,
  connectionString: config.postgresUrl,
});
