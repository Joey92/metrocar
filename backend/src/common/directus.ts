import axios from 'axios';
import { DIRECTUS_URL, DIRECTUS_USER_TOKEN } from './config';

export const directus = axios.create({
  baseURL: DIRECTUS_URL,
  headers: {
    Authorization: `Bearer ${DIRECTUS_USER_TOKEN}`,
  },
});

export default directus;
