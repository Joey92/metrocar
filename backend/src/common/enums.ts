export enum Messages {
  USER_EXISTS = 'user_exists',
  FIELD_REQUIRED = 'field_required',
  FIELD_INVALID = 'field_invalid',
  ROUTER_UNAVAILABLE = 'router_unavailable',
  REVERSE_GEOCODE_UNAVAILABLE = 'reverse_geocode_unavailable',

  NO_STOPS_ON_ROUTE = 'no_stops_on_route',

  INTERNAL_SERVER_ERROR = 'internal_server_error',

  PLACE_NOT_FOUND = 'place_not_found',

  USERNAME_INVALID = 'username_invalid',
  EMAIL_INVALID = 'email_invalid',
  EMAIL_REGISTERED = 'email_registered',

  ACCOUNT_CREATED = 'account_created',

  LOGIN_FAILED = 'login_failed',

  GENERIC_ERROR = 'generic_error',
}
