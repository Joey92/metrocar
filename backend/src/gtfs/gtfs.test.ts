import { db } from '../common/db';
import { getGTFSCSV } from './gtfs';
import { deleteTestDataQuery, testDataQuery } from '../test_helper/helpers';

beforeAll(() => db.none(testDataQuery));
afterAll(() => db.none(deleteTestDataQuery));

it('makes gtfs correctly', async () => {
  const data = await getGTFSCSV();

  data.forEach((csv) => {
    expect(csv.content).toMatchSnapshot(csv.name);
  });
});
