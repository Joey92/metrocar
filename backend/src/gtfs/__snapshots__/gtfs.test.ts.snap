// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makes gtfs correctly: agency 1`] = `
"\\"agency_id\\",\\"agency_name\\",\\"agency_url\\",\\"agency_timezone\\",\\"agency_lang\\",\\"agency_phone\\",\\"agency_fare_url\\",\\"agency_email\\",\\"created_at\\",\\"updated_at\\"
\\"8457aec4-c852-4f84-9dc0-e81e79555683\\",\\"MetroCar\\",\\"http://metrocar.org\\",\\"Europe/Berlin\\",,,,,\\"1633758743719\\",
"
`;

exports[`makes gtfs correctly: calendar 1`] = `
"\\"service_id\\",\\"monday\\",\\"tuesday\\",\\"wednesday\\",\\"thursday\\",\\"friday\\",\\"saturday\\",\\"sunday\\",\\"start_date\\",\\"end_date\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"1\\",\\"1\\",\\"1\\",\\"1\\",\\"1\\",\\"0\\",\\"0\\",\\"20211011\\",\\"20211029\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"1\\",\\"1\\",\\"1\\",\\"1\\",\\"1\\",\\"0\\",\\"0\\",\\"20211011\\",\\"20211029\\"
"
`;

exports[`makes gtfs correctly: calendar_dates 1`] = `
"\\"service_id\\",\\"date\\",\\"exception_type\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"20211011\\",\\"1\\"
"
`;

exports[`makes gtfs correctly: feed_info 1`] = `
"\\"feed_publisher_name\\",\\"feed_publisher_url\\",\\"feed_lang\\",\\"feed_start_date\\",\\"feed_end_date\\",\\"feed_version\\",\\"feed_contact_email\\"
\\"MetroCar\\",\\"http://metrocar.org\\",\\"de\\",\\"20210101\\",\\"20211231\\",\\"1\\",\\"contact@metrocar.org\\"
"
`;

exports[`makes gtfs correctly: routes 1`] = `
"\\"route_id\\",\\"agency_id\\",\\"route_short_name\\",\\"route_long_name\\",\\"route_desc\\",\\"route_type\\",\\"route_url\\",\\"route_color\\",\\"route_text_color\\",\\"continuous_pickup\\",\\"continuous_drop_off\\",\\"created_at\\",\\"updated_at\\"
\\"87215dff60e31493d1b37504f6c86a5f80c2d24e\\",\\"8457aec4-c852-4f84-9dc0-e81e79555683\\",\\"L1\\",\\"From Ahrensfelde Bahnhof to Sozialgericht\\",\\"From Ahrensfelde Bahnhof to Sozialgericht with 16 stops.\\",\\"3\\",,,,\\"3\\",\\"3\\",\\"1633758439450\\",
\\"4942bed018bb4e9d284b9a09005fb8dec4ac919c\\",\\"8457aec4-c852-4f84-9dc0-e81e79555683\\",\\"L3\\",\\"From Invalidenstrasse to Dorfstrasse/Merow Str.\\",\\"From Invalidenstrasse to Dorfstrasse/Merow Str. with 15 stops.\\",\\"3\\",,,,\\"3\\",\\"3\\",\\"1633758603363\\",
"
`;

exports[`makes gtfs correctly: shapes 1`] = `
"\\"shape_id\\",\\"shape_pt_lat\\",\\"shape_pt_lon\\",\\"shape_pt_sequence\\",\\"shape_dist_traveled\\"
"
`;

exports[`makes gtfs correctly: stop_times 1`] = `
"\\"trip_id\\",\\"arrival_time\\",\\"departure_time\\",\\"stop_id\\",\\"stop_sequence\\",\\"stop_headsign\\",\\"shape_dist_traveled\\",\\"pickup_type\\",\\"drop_off_type\\",\\"continuous_pickup\\",\\"continuous_drop_off\\",\\"timepoint\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"07:46:00\\",\\"07:47:00\\",\\"ca75e329-6063-426a-8555-1f2d42b1db8d\\",\\"1\\",,\\"0\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"07:48:00\\",\\"07:49:00\\",\\"c3794997-1768-4fd6-a5a8-cf6a57ee9ab9\\",\\"2\\",,\\"1\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"07:50:00\\",\\"07:51:00\\",\\"7e5848ed-3e58-4a7b-9f53-cd661618c473\\",\\"3\\",,\\"1\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"07:52:00\\",\\"07:53:00\\",\\"70921547-1ee2-4e5b-a1a9-16de5d6af2a0\\",\\"4\\",,\\"2\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"07:56:00\\",\\"07:57:00\\",\\"56fa7594-f5dc-4ec8-8b04-915f86e13acf\\",\\"5\\",,\\"4\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"07:58:00\\",\\"07:59:00\\",\\"732301c6-d345-4260-b562-b32cdf5154ea\\",\\"6\\",,\\"5\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:01:00\\",\\"08:02:00\\",\\"65e41ebc-5fdb-4b5a-bdcb-a36d3585cf8e\\",\\"7\\",,\\"6\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:08:00\\",\\"08:09:00\\",\\"1a19d4ba-3802-4f20-9ad9-a808de518645\\",\\"8\\",,\\"11\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:11:00\\",\\"08:12:00\\",\\"a085b7a1-a366-4e85-ae87-f8c2c43c7e16\\",\\"9\\",,\\"12\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:12:00\\",\\"08:13:00\\",\\"75dda211-e938-4a9e-b228-f796ff042f19\\",\\"10\\",,\\"12\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:14:00\\",\\"08:15:00\\",\\"cd3ea1ac-1e8c-4e60-b948-e7ab3f6b9ce0\\",\\"11\\",,\\"13\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:17:00\\",\\"08:18:00\\",\\"6c7a3c6c-69f4-4808-99cd-90db43cd9f11\\",\\"12\\",,\\"14\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:19:00\\",\\"08:20:00\\",\\"36bc1964-c0f1-44ea-9bc4-3a44be7f6796\\",\\"13\\",,\\"15\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:20:00\\",\\"08:21:00\\",\\"0298db74-42f6-44e2-9bf3-29914e99a3bd\\",\\"14\\",,\\"15\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:22:00\\",\\"08:23:00\\",\\"02ca1202-4f8c-45ac-ae4a-9607a7385bc5\\",\\"15\\",,\\"16\\",,,,,\\"0\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"08:30:00\\",\\"08:31:00\\",\\"3fef10f7-9c1d-4724-beda-a571064eedaa\\",\\"16\\",,\\"19\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"07:48:00\\",\\"07:49:00\\",\\"ca75e329-6063-426a-8555-1f2d42b1db8d\\",\\"1\\",,\\"0\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"07:50:00\\",\\"07:51:00\\",\\"c3794997-1768-4fd6-a5a8-cf6a57ee9ab9\\",\\"2\\",,\\"1\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"07:52:00\\",\\"07:53:00\\",\\"7e5848ed-3e58-4a7b-9f53-cd661618c473\\",\\"3\\",,\\"1\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"07:54:00\\",\\"07:55:00\\",\\"70921547-1ee2-4e5b-a1a9-16de5d6af2a0\\",\\"4\\",,\\"2\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"07:58:00\\",\\"07:59:00\\",\\"56fa7594-f5dc-4ec8-8b04-915f86e13acf\\",\\"5\\",,\\"4\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:00:00\\",\\"08:01:00\\",\\"732301c6-d345-4260-b562-b32cdf5154ea\\",\\"6\\",,\\"5\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:03:00\\",\\"08:04:00\\",\\"65e41ebc-5fdb-4b5a-bdcb-a36d3585cf8e\\",\\"7\\",,\\"6\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:10:00\\",\\"08:11:00\\",\\"1a19d4ba-3802-4f20-9ad9-a808de518645\\",\\"8\\",,\\"11\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:13:00\\",\\"08:14:00\\",\\"a085b7a1-a366-4e85-ae87-f8c2c43c7e16\\",\\"9\\",,\\"12\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:14:00\\",\\"08:15:00\\",\\"75dda211-e938-4a9e-b228-f796ff042f19\\",\\"10\\",,\\"12\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:16:00\\",\\"08:17:00\\",\\"cd3ea1ac-1e8c-4e60-b948-e7ab3f6b9ce0\\",\\"11\\",,\\"13\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:19:00\\",\\"08:20:00\\",\\"6c7a3c6c-69f4-4808-99cd-90db43cd9f11\\",\\"12\\",,\\"14\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:21:00\\",\\"08:22:00\\",\\"36bc1964-c0f1-44ea-9bc4-3a44be7f6796\\",\\"13\\",,\\"15\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:22:00\\",\\"08:23:00\\",\\"0298db74-42f6-44e2-9bf3-29914e99a3bd\\",\\"14\\",,\\"15\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:24:00\\",\\"08:25:00\\",\\"02ca1202-4f8c-45ac-ae4a-9607a7385bc5\\",\\"15\\",,\\"16\\",,,,,\\"0\\"
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"08:32:00\\",\\"08:33:00\\",\\"3fef10f7-9c1d-4724-beda-a571064eedaa\\",\\"16\\",,\\"19\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"23:48:00\\",\\"23:49:00\\",\\"f92ebae7-a890-45db-9069-81031ccf2bb6\\",\\"1\\",,\\"0\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"23:50:00\\",\\"23:51:00\\",\\"4d28f5a5-61e0-4978-b623-c2635e4aea77\\",\\"2\\",,\\"1\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"23:58:00\\",\\"23:59:00\\",\\"0f57ded1-eb6d-45d4-a966-6d6da57d0d9e\\",\\"3\\",,\\"5\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:00:00\\",\\"24:01:00\\",\\"68a346df-b68b-4949-8158-becb3384f07e\\",\\"4\\",,\\"6\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:02:00\\",\\"24:03:00\\",\\"cd3ea1ac-1e8c-4e60-b948-e7ab3f6b9ce0\\",\\"5\\",,\\"7\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:04:00\\",\\"24:05:00\\",\\"75efc0b9-2a8c-4704-bdaf-7bea0acc5054\\",\\"6\\",,\\"7\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:06:00\\",\\"24:07:00\\",\\"16b98842-b279-4f7d-82bf-97a6dbe941c2\\",\\"7\\",,\\"8\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:08:00\\",\\"24:09:00\\",\\"0b131d1c-6da5-4eb1-be85-88a96287651e\\",\\"8\\",,\\"9\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:13:00\\",\\"24:14:00\\",\\"caa3cf04-73a1-4ea0-935a-6b17142a2ee3\\",\\"9\\",,\\"12\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:16:00\\",\\"24:17:00\\",\\"9216e821-03d2-4ed5-a49d-02211d2b2983\\",\\"10\\",,\\"13\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:19:00\\",\\"24:20:00\\",\\"b5cb7197-ab79-4257-960f-cc5797b9fb65\\",\\"11\\",,\\"15\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:22:00\\",\\"24:23:00\\",\\"480158dc-cf20-44d7-bb2f-2583701eb039\\",\\"12\\",,\\"16\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:24:00\\",\\"24:25:00\\",\\"220ac85e-25fd-4caa-a464-f645b4d70804\\",\\"13\\",,\\"17\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:29:00\\",\\"24:30:00\\",\\"6c3a80c7-1ae9-41b2-96bf-0fb4e39db61a\\",\\"14\\",,\\"19\\",,,,,\\"0\\"
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"24:33:00\\",\\"24:34:00\\",\\"8dc50edc-df60-432f-a9df-0c5ae961c36e\\",\\"15\\",,\\"20\\",,,,,\\"0\\"
"
`;

exports[`makes gtfs correctly: stops 1`] = `
"\\"stop_id\\",\\"stop_code\\",\\"stop_name\\",\\"tts_stop_name\\",\\"stop_desc\\",\\"stop_lat\\",\\"stop_lon\\",\\"zone_id\\",\\"stop_url\\",\\"location_type\\",\\"parent_stop_id\\",\\"stop_timezone\\",\\"wheelchair_boarding\\",\\"level_id\\",\\"platform_code\\",\\"active\\",\\"created_at\\",\\"updated_at\\"
\\"32af8a7d-c168-4436-bae2-4b3382704879\\",,\\"Danziger Strasse / Greifswalder strasse\\",,,\\"52.53658\\",\\"13.43245\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"0f57ded1-eb6d-45d4-a966-6d6da57d0d9e\\",,\\"Danziger Strasse / Greifswalder strasse\\",,,\\"52.53559\\",\\"13.4338\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"68a346df-b68b-4949-8158-becb3384f07e\\",,\\"Danziiger Strasse / Kniprodestrasse\\",,,\\"52.53236\\",\\"13.44057\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"16b98842-b279-4f7d-82bf-97a6dbe941c2\\",,\\"Vienna House\\",,,\\"52.52849\\",\\"13.45691\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"b5cb7197-ab79-4257-960f-cc5797b9fb65\\",,\\"Le Prom\\",,,\\"52.5422\\",\\"13.54172\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"220ac85e-25fd-4caa-a464-f645b4d70804\\",,\\"Lidl\\",,,\\"52.55684\\",\\"13.55427\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"97991102-d281-4f31-8c48-6315a15002af\\",,\\"Parkplatz\\",,,\\"52.67747\\",\\"13.39005\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"66fe7d7e-baf7-47a1-b897-3869abde022f\\",,\\"Parkplatz\\",,,\\"52.67669\\",\\"13.39455\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"2a021402-b36f-4249-b624-a00b03f9e60a\\",,\\"Autogas Blumberg\\",,,\\"52.60337\\",\\"13.60977\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"e4edb865-c4dc-40cb-8cf8-bfad17105862\\",,\\"Norma Blumberg\\",,,\\"52.59627\\",\\"13.59973\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"97494ca2-32a2-4760-8937-24dbd30619ba\\",,\\"Apfelsteig\\",,,\\"52.58053\\",\\"13.58057\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"f119025f-5d55-48ee-a6c7-812bafc8d13b\\",,\\"Ahrensfelder Dreieck\\",,,\\"52.57885\\",\\"13.58244\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"8dc50edc-df60-432f-a9df-0c5ae961c36e\\",,\\"Dorfstrasse/Merow Str.\\",,,\\"52.57685\\",\\"13.58267\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"c3794997-1768-4fd6-a5a8-cf6a57ee9ab9\\",,\\"Rathaus Ahrensfelde\\",,,\\"52.5758\\",\\"13.57568\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"93bebb1a-941f-4921-9a7d-52ff24ca70e3\\",,\\"Ahrensfelder Strasse\\",,,\\"52.60292\\",\\"13.5305\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"8104a4ab-62af-4b56-925f-4f0c57cff32b\\",,\\"Karl Marx Str.\\",,,\\"52.60261\\",\\"13.52812\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"1b868f6e-bcff-4491-87b6-1cf788f3c03d\\",,\\"Shell Tankstelle\\",,,\\"52.60121\\",\\"13.5147\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"3b78a161-1b08-41e9-ad71-fa7e17e35358\\",,\\"Lindenzwerge\\",,,\\"52.60225\\",\\"13.53053\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"eaaf40b5-be40-4576-b560-f267ee79b4b3\\",,\\"Ahrensfelde Liindenberger Str.\\",,,\\"52.58324\\",\\"13.56724\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"ca75e329-6063-426a-8555-1f2d42b1db8d\\",,\\"Ahrensfelde Bahnhof\\",,,\\"52.58135\\",\\"13.57418\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"1d68e39a-3618-4fc9-a588-4b0af4b66b5b\\",,\\"Ahrensfelde Bahnhof\\",,,\\"52.58019\\",\\"13.5731\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"9216e821-03d2-4ed5-a49d-02211d2b2983\\",,\\"Total Tankstelle\\",,,\\"52.53558\\",\\"13.52253\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"0b131d1c-6da5-4eb1-be85-88a96287651e\\",,\\"Netto Landsberger allee\\",,,\\"52.53106\\",\\"13.46881\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"6c7a3c6c-69f4-4808-99cd-90db43cd9f11\\",,\\"Friedenstrasse\\",,,\\"52.52356\\",\\"13.4331\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"36bc1964-c0f1-44ea-9bc4-3a44be7f6796\\",,\\"Netto Mollstrasse\\",,,\\"52.52343\\",\\"13.42606\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"0cf55eaf-62d6-403b-8a5d-8f81542fcb0f\\",,\\"Mollstrasse\\",,,\\"52.52433\\",\\"13.42141\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"0298db74-42f6-44e2-9bf3-29914e99a3bd\\",,\\"Mollstrasse\\",,,\\"52.52488\\",\\"13.42139\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"0dfd4584-9fb0-41bd-a66b-043c3aa35ae3\\",,\\"Otto braun strasse\\",,,\\"52.52367\\",\\"13.41881\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"aa5ff2d0-da2d-4c27-9a3e-cac620d9c3ea\\",,\\"Memhardstrasse\\",,,\\"52.52384\\",\\"13.41105\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"5d734ed6-03dc-4c3c-886d-6fa4d379f55f\\",,\\"Memhardstrasse\\",,,\\"52.52405\\",\\"13.411\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"872ee76e-b2cf-4cb1-98a2-2db59c630612\\",,\\"Spandauert Strasse\\",,,\\"52.51892\\",\\"13.40618\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"81371c85-b1d9-40fa-97ff-378964ffee38\\",,\\"Am Lustgarten\\",,,\\"52.51869\\",\\"13.4007\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"3217ad8e-6c74-47a7-a35c-1c439528238b\\",,\\"Brandenburger Tor\\",,,\\"52.51664\\",\\"13.38026\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"7796c194-4527-428e-a0be-60b3681ecac6\\",,\\"Hotel Adlon\\",,,\\"52.51626\\",\\"13.38031\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"4d28f5a5-61e0-4978-b623-c2635e4aea77\\",,\\"Platz vor dem neuen Tor\\",,,\\"52.5286\\",\\"13.37842\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"3fef10f7-9c1d-4724-beda-a571064eedaa\\",,\\"Sozialgericht\\",,,\\"52.52755\\",\\"13.37184\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"f92ebae7-a890-45db-9069-81031ccf2bb6\\",,\\"Invalidenstrasse\\",,,\\"52.52723\\",\\"13.3727\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"f58f4701-2c19-4398-ba1f-56622e70a5e3\\",,\\"Karl Liebknecht Strasse\\",,,\\"52.52637\\",\\"13.4145\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"47d895e8-41d2-4346-8504-055b72998f87\\",,\\"Park  Inn Hoitel\\",,,\\"52.52299\\",\\"13.41348\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"8d0f7a72-c8c7-4002-badd-bf4b2bccd333\\",,\\"Alte Schoenhauser strasse\\",,,\\"52.52512\\",\\"13.40649\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"5f446401-3efc-46a5-a2cf-ebd2525fd60b\\",,\\"Alte Schoenhauser strasse\\",,,\\"52.52511\\",\\"13.40656\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"02ca1202-4f8c-45ac-ae4a-9607a7385bc5\\",,\\"Torstrasse\\",,,\\"52.52744\\",\\"13.41534\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"75efc0b9-2a8c-4704-bdaf-7bea0acc5054\\",,\\"SEZ\\",,,\\"52.52655\\",\\"13.44936\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"cd3ea1ac-1e8c-4e60-b948-e7ab3f6b9ce0\\",,\\"SEZ\\",,,\\"52.52624\\",\\"13.44634\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"75dda211-e938-4a9e-b228-f796ff042f19\\",,\\"Burgerking Landsberger Allee\\",,,\\"52.5289\\",\\"13.45658\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"a085b7a1-a366-4e85-ae87-f8c2c43c7e16\\",,\\"Landsberger Allee\\",,,\\"52.53028\\",\\"13.46249\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"1a19d4ba-3802-4f20-9ad9-a808de518645\\",,\\"City Hotel Landsberger Allee\\",,,\\"52.5334\\",\\"13.47778\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"caa3cf04-73a1-4ea0-935a-6b17142a2ee3\\",,\\"IKEA Lichtenberg\\",,,\\"52.53453\\",\\"13.51267\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"480158dc-cf20-44d7-bb2f-2583701eb039\\",,\\"JET Tankstelle\\",,,\\"52.55173\\",\\"13.54973\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"65e41ebc-5fdb-4b5a-bdcb-a36d3585cf8e\\",,\\"SBahn Marzahn\\",,,\\"52.54391\\",\\"13.54198\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"732301c6-d345-4260-b562-b32cdf5154ea\\",,\\"Sbahn Roul-Wallenberg-Str.\\",,,\\"52.55081\\",\\"13.54804\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"56fa7594-f5dc-4ec8-8b04-915f86e13acf\\",,\\"SBahn Mehrower Allee\\",,,\\"52.55798\\",\\"13.55423\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"6c3a80c7-1ae9-41b2-96bf-0fb4e39db61a\\",,\\"Amt\\",,,\\"52.57282\\",\\"13.56794\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"70921547-1ee2-4e5b-a1a9-16de5d6af2a0\\",,\\"Ahrensfelde Sbahn\\",,,\\"52.57159\\",\\"13.56618\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
\\"7e5848ed-3e58-4a7b-9f53-cd661618c473\\",,\\"Edeka ahrensfelde\\",,,\\"52.57492\\",\\"13.57284\\",,,,,,\\"1\\",,,,\\"1633292430839\\",
"
`;

exports[`makes gtfs correctly: trips 1`] = `
"\\"trip_id\\",\\"route_id\\",\\"service_id\\",\\"trip_headsign\\",\\"trip_short_name\\",\\"direction_id\\",\\"shape_id\\",\\"block_id\\",\\"wheelchair_accessible\\",\\"bikes_allowed\\",\\"start_date\\",\\"owner\\",\\"created_at\\",\\"updated_at\\"
\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"87215dff60e31493d1b37504f6c86a5f80c2d24e\\",\\"404f687c-1bf2-4eab-a91b-28feac8b8abf\\",\\"Sozialgericht\\",\\"BAR IZ 36E\\",,\\"87215dff60e31493d1b37504f6c86a5f80c2d24e\\",,,,\\"1633903200000\\",\\"2ea805d0-19d1-46fe-8dd6-70285f02a23a\\",\\"1633758439450\\",
\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"87215dff60e31493d1b37504f6c86a5f80c2d24e\\",\\"2c8c01d5-6d9d-47b8-bd9d-8eff870aeff5\\",\\"Sozialgericht\\",\\"BAR IZ 36E\\",,\\"87215dff60e31493d1b37504f6c86a5f80c2d24e\\",,,,,\\"2ea805d0-19d1-46fe-8dd6-70285f02a23a\\",\\"1633758485175\\",
\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"4942bed018bb4e9d284b9a09005fb8dec4ac919c\\",\\"04efdcd7-3bfc-4ab4-aee0-dbeb0f326bae\\",\\"Dorfstrasse/Merow Str.\\",\\"BAR IZ 36E\\",,\\"4942bed018bb4e9d284b9a09005fb8dec4ac919c\\",,,,,\\"2ea805d0-19d1-46fe-8dd6-70285f02a23a\\",\\"1633758603363\\",
"
`;
